src/ontology/cl-edit.owl

```diff
@@ -32727,6 +32729,26 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:7107724") oboInOwl:hasRe
 AnnotationAssertion(rdfs:label obo:CL_4052011 "follicle associated enterocyte")
 SubClassOf(obo:CL_4052011 obo:CL_0000584)
 
+# Class: obo:CL_4052015 (endocrine gland capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18480313") obo:IAO_0000115 obo:CL_4052015 "Any capillary endothelial cell that is part of the endocrine gland.")
+AnnotationAssertion(terms:contributor obo:CL_4052015 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052015 "2024-09-30T15:27:49Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4052015 "endocrine gland capillary endothelial cell")
+SubClassOf(obo:CL_4052015 obo:CL_0002144)
+SubClassOf(obo:CL_4052015 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002368))
+
+# Class: obo:CL_4052016 (pituitary gland capillary endothelial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10810312") Annotation(oboInOwl:hasDbXref "PMID:32910242") Annotation(oboInOwl:hasDbXref "PMID:4899902") obo:IAO_0000115 obo:CL_4052016 "A capillary endothelial cell that is part of the pituitary gland, particularly in the anterior pituitary. This cell is distinguished by its numerous fenestrations, which are covered by a single-layered diaphragm. The diaphragm features a dense ring-like structure, facilitating the passage of hormones and other substances essential for endocrine signalling.")
```

Comment:
Why particularly in the anterior pituitary - are there just more capillaries there?  Name+  logical def defines as Any capillary endothelial cell of the pituitary gland, so needs to not be defining as subset.  Please clarify or drop detail.

Reply:
I have changed the definition to make it more inclusive, as both the anterior and posterior lobes of the pituitary gland have fenestrated capillaries.

For the anterior lobe, I found a paper that provides detailed information about the fenestrations specifically within the capillary endothelial cells of the anterior pituitary in rats. I have included these information in the comments section but wasn't sure whether I should include it in the extended description instead